packages/c/sshnpd/src/handle_ssh_request.c

```diff
@@ -4,13 +4,14 @@
 #include <atchops/base64.h>
 #include <atchops/iv.h>
 #include <atchops/rsa_key.h>
+#include <atclient/cjson.h>
```

Comment:
You will see me point to our wrapped cjson built into atclient, this header exports cjson properly based on how atclient is built